### PR TITLE
Add r2d2 part.

### PR DESCRIPTION
**Pull request by pchaya**

### CODE REVIEW

**Comment by pchaya** [comment]
@mvincerx

**Comment by pchaya** [comment]
@aselker-2 hey can you review this?

**Comment by aselker-2** [comment]
i'm pretty sure he *slides*

File: starwars-ipsum.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ she's got a lot of spirit. I don't know, what do you think? What!?
 - No! Alderaan is peaceful. We have no weapons. You can't possibly…
 - I find your lack of faith disturbing.
 
+Then R2D2 glided  in.
```
Comment (aselker-2) [inline starwars-ipsum.md]: s/glided/slid

Comment (pchaya) [reply starwars-ipsum.md]: semantics..

**Comment by mvincerx** [comment]
Wait! Jedis knights must always return to impeccability of their word! What change must I make?!

**Pull request merged**